### DISCUSSION TITLE
Add Functionality to sign in to SelfHosted Outline instance via Mac Client

### DISCUSSION THREAD

**Discussion by Squiddim** [outline/outline#6685] [Self Hosting]
Currently there is no option to sign in to a Selfhosted Instance via the Mac Client.
I for example use Keycloak as my Authentication Provider and there is no Setting to point the Client to my Instance

**Comment by iammattmartin** [comment]
I asked this recently and was advised not yet possible but support may be added in the future if there is enough demand.

**Comment by eurek** [comment]
Hello!
Up ! It would be great to use Mac client wih self hosted instance.

**Comment by levifig** [comment]
AFAIK the "Mac Client" is just an Electron wrapper to the web interface. You can achieve similar results by opening your self-hosted instance on Safari and doing `File -> Add to Dock...".

Am I missing something?
